synapse/events/utils.py

```diff
@@ -100,22 +100,18 @@ def format_event_raw(d):
 
 
 def format_event_for_client_v1(d):
-    d["user_id"] = d.pop("sender", None)
+    d = format_event_for_client_v2(d)
+
+    d["user_id"] = d.get("sender", None)
```

Comment:
Should this only set if it's present? I don't like the number of random nulls we already add to responses...